Slim/Plugin/ExtendedBrowseModes/Settings.pm

```diff
@@ -16,6 +16,7 @@ use Slim::Utils::Log;
 use Slim::Utils::Misc;
 use Slim::Utils::Strings qw(string);
 use Slim::Utils::Prefs;
+use List::Util qw(max);
```

Comment:
Please move this before the project own imports. I like to have `Slim` and `Plugins` modules separate from the other imports.